source/agora/consensus/EnrollmentManager.d

```diff
@@ -664,6 +667,77 @@ public class EnrollmentManager
         return false;
     }
 
+    /***************************************************************************
+
+        Get the key for the enrollment for this node
+
+        If getting an enrollment key fails, this displays a log messge and
+        returns `Hash.init`. If the database operation fails, it means that node
+        is not in the normal situation overall.
```

Comment:
"the node is not in the normal situation" is not quite a good explanation. If we detect and error situation, we usually try to remedy it. In this code, I was assuming that the failure was "normal" in case the UTXO wasn't propagated yet. But that doesn't seem to be the case ?

Reply:
Looking at the code again, when the node is not enrolled, I assume the `SELECT` will return an empty result, so no exception will be thrown ? If so that should be documented.